src/platform/apollolake/include/platform/lib/dma.h

```diff
@@ -26,6 +26,7 @@
 #define DMA_HOST_OUT_DMAC	5
 #define DMA_LINK_IN_DMAC	6
 #define DMA_LINK_OUT_DMAC	7
+#define DMA_BUF_COPIER		8
 
```

Comment:
Same here, this looks misaligned.

Reply:
@dbaluta Same as the above, it's just GH display issue.